### PR TITLE
Re-Add color overlay

### LINKED ISSUE #288: Tab Controllers:  Update Rate Frame Rate independent.
We need to adjust our ticks to execute based upon frame rate since there is such a huge discrepancy between 80hz and 144 hz.

WE should on initilization be able to read in frame rate with `k_pch_Null_DisplayFrequency_Float ` or if that doesn't work we should manually grab hmd names and hard code in. (with default to 90).

If we compute the change during initialization there will be no overly large performance loss aside from slightly higher memory usage from another member variable for each tab controller.

### PR DESCRIPTION
**Pull request by ykeara**
fixes #288 
update rates for tabcontrollers now scale based on reported value to steamvr falls back to assuming 90 if there is an error getting the value.

Fixed an error w/ the chaperone toggle being in a visually wrong state on start-up

Re-added the color overlay method as an option for headsets that don't support gain.

### CODE REVIEW

File: src/res/qml/video_page/overlaycolorbright/OverlayMethodGroupBox.qml [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+        }
+
+        RowLayout {
+            Layout.fillWidth: true
+
+            MyToggleButton {
+                id: brightnessToggle
+                text: "Toggle On/Off"
+                onCheckedChanged: {
+                    VideoTabController.setBrightnessEnabled(this.checked, true)                }
```
Comment (username223) [inline src/res/qml/video_page/overlaycolorbright/OverlayMethodGroupBox.qml]: Missing newline for the }?

File: src/tabcontrollers/ChaperoneTabController.h [hunk 2]
```diff
@@ -170,6 +171,8 @@ class ChaperoneTabController : public QObject
 
     int m_updateTicksChaperoneReload = 0;
 
+    unsigned int m_k_chaperoneSettingsUpdateCounter = 101;
```
Comment (username223) [inline src/tabcontrollers/ChaperoneTabController.h]: Why is it prefixed with `m_` and `k_` at the same time?

File: src/tabcontrollers/ChaperoneTabController.cpp [hunk 3]
```diff
@@ -638,7 +645,7 @@ void ChaperoneTabController::eventLoopTick(
         }
     }
 
-    if ( settingsUpdateCounter >= k_chaperoneSettingsUpdateCounter )
+    if ( settingsUpdateCounter >= m_k_chaperoneSettingsUpdateCounter )
```
Comment (username223) [inline src/tabcontrollers/ChaperoneTabController.cpp]: Why is it prefixed with `m_` and `k_` at the same time?

File: src/tabcontrollers/ChaperoneTabController.h (hunk 2 above)
Comment (ykeara) [reply src/tabcontrollers/ChaperoneTabController.h]: It's still a key, its just on initialization we adjust this value based on `k_chaperoneSettingsUpdateCounter` and adjust based on `k_pch_SteamVR_PreferredRefreshRate` thus its a member variable key. As for exact naming decision it makes it easy to find the parent key.

That being said adjusting it in each tab-controller might not be best option.

File: src/tabcontrollers/ChaperoneTabController.cpp (hunk 3 above)
Comment (ykeara) [reply src/tabcontrollers/ChaperoneTabController.cpp]: see above

File: src/res/qml/video_page/overlaycolorbright/OverlayMethodGroupBox.qml (hunk 1 above)
Comment (ykeara) [reply src/res/qml/video_page/overlaycolorbright/OverlayMethodGroupBox.qml]: yep

**Pull request merged**